config/initializers/google_big_query.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require "google/cloud/bigquery"
+require "base64"
+
+Google::Cloud::Bigquery.configure do |config|
+  config.project_id = "ecf-bq"
```

Comment:
Do we want to put this in its own BigQuery project rather than the ECF one?

Reply:
Think `ecf-bq` project is the only one available to us to use.